server/views/applications/view.njk

```diff
@@ -132,12 +131,12 @@
               JavaScript domains
             </dt>
             <dd class="govuk-summary-list__value">
-              {{ key.js_domains }}
+              {{ key.jsDomains }}
```

Comment:
what does this display look like if you have multiple js domains?

Reply:
We could iterate over the list with something like 
```
{% for domain in key.jsDomains %}
   domain
{% endblock %}
```

Reply:
yeah, that's what I've got in mine.

Reply:
But out of interest, what does it look like if you have multiple displays as is? Does it all display in one line?

Reply:
Currently the display has the js domains like so
<img width="1680" alt="Screenshot 2020-10-28 at 15 42 09" src="https://user-images.githubusercontent.com/65958266/97460877-11f28680-1935-11eb-8f68-1afbcaa6ce10.png">

Reply:
it is  separated with a comma like the instructions tell the user to do

Reply:
whereas multiple ip addresses display on multiple lines. Which one is preferable? the comma separation is how the user is instructed to input, but I'm not sure that it necessarily needs to display like that.....

Reply:
Prototype is unclear, multiple lines is what I did with IPs however we may want to speak to Emma.

Reply:
If nothing else I'd like to see a space between the inputs to make them more readable rather than a long run on string

Reply:
I don't think the prototype has an example

Reply:
I messaged Emma to see how it should be displayed and then I will push up the changes